.github/workflows/check.yml

```diff
@@ -0,0 +1,28 @@
+name: Static Analysis
+
+on:
+  pull_request:
+    branches:
+     - 3.*-devel
+     - master
+  push:
+    branches:
+     - 3.*-devel
+     - master
+  schedule:
+    - cron: 0 0 * * 0
```

Comment:
I'm not sury why run this regularly too? You should have 100% coverage with the triggers above. Also, it won't run on the 3.*-devel kind of branch.

I guess the cron trigger would be useful for external changes that can break the checks regardless of code?

Reply:
>Also, it won't run on the 3.*-devel kind of branch.

If I understand it correctly it will run on the default branch, which should be enough.

> I guess the cron trigger would be useful for external changes that can break the checks regardless of code?

Yes, that's the general idea. It's probably not as important for blivet which is a relatively active project, but for some of our less active projects where there could be few weeks between patches (like blivet-gui where I added this action too) and I'd like to be notified when something goes wrong sooner than later. With pylint this mostly means new warnings that were detect by a new pylint (or astroid) version. It's always annoying to discover that pylint started complaining about something completely unrelated to my PR just because there was a new version few weeks ago.

Reply:
Ah, ok, that does make sense.